lib/explorer/data_frame.ex

```diff
@@ -516,7 +516,7 @@ defmodule Explorer.DataFrame do
 
     * `:max_rows` - Maximum number of lines to read. (default: `nil`)
 
-    * `:null_character` - The string that should be interpreted as a nil value. (default: `"NA"`)
+    * `:null_character` - The string that should be interpreted as a nil value. (default: `nil`)
```

Comment:
We could also rename the option to `:nil_values` and accept a list of values. 

```suggestion
    * `:nil_values` - A string or a list of strings that should be interpreted as nil values.
       By default, all empty cells are nil, but it's common to have nil values declared as "NA",
       so this option is useful for that. (default: `nil`) 
```

Reply:
Read my mind :)

Only clarification needed is I interpret "A string or a list of strings" as list | string | nil. Is that correct? I could just assume list | nil for now.

Reply:
Yeah, it was correct. But I think we can follow José's suggestion and only accept `string | list(string)` for now. The default would be `[]`. I think we can still treat this as an option down in the NIF. WDYT?